docs/patterns/geodes.md

```diff
@@ -81,4 +91,6 @@ This pattern might not be suitable for
 ## Examples
 
 - Windows Active Directory implements an early variant of this pattern. Multi-primary replication means all updates and requests can in theory be served from all serviceable nodes, but FSMO roles mean that all geodes aren't equal.
+- A [geode-pattern-accelerator](https://github.com/lucashuet93/geode-pattern-accelerator) application on GitHub showcases this design pattern in practice and is designed to help developers implement it with real-world APIs.
+- The [globally distributed applications using Cosmos DB](https://docs.microsoft.com/en-us/azure/architecture/solution-ideas/articles/globally-distributed-mission-critical-applications-using-cosmos-db) article examines a geographical based deployment that utilizes Traffic Manager for load balancing and Azure App Service to host the API code.
```

Comment:
Ideally we should add a link back from that doc to here as well.

Reply:
Opened a [PR on that document ](https://github.com/MicrosoftDocs/architecture-center/pull/2865) as well that links back here